content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,329 @@
+.. _MT_N_layered_Earth:
+
+Response of a layered Earth to a plane wave
+===========================================
+
+.. topic:: Purpose
+
+    We present here a 1D modelisation of the MagnetoTelluric waves in a layered Earth and the associated geophysical data. Our goal is to build better representation of the different physical phenomenons and better understanding of the resulting data.
```

Comment:
Nice, I like the addition of the purpose. I think we can make it stronger - my thinking here is 
- why present a 1D model? (first we need the motivation of examining the physical processes of plane wave propagation through the earth) 
- "different physical phenomenons" is a bit vague - we want to choose language that tells us the scope of the page

From this, I am thinking something along the lines of:
`Plane wave propagation through the earth is at the core of the Magnetotelluric problem. Here, we investigate the physics of plane wave propagation through a 1D layered earth and discuss the connection between the physical responses and what we observe in MT data`

Feel free to adapt this to your style!

Also: spell check :) . In sublime: 
<img width="1199" alt="mt_n_layered_earth rst em 2016-04-09 09-49-56" src="https://cloud.githubusercontent.com/assets/6361812/14404972/8a2f5f64-fe38-11e5-8ec0-ef4e83e978f5.png">